test/compile/mark/tick.test.ts

```diff
@@ -66,7 +66,7 @@ describe('Mark: Tick', () => {
       expect(props.xc).toEqual({scale: X, field: 'Horsepower'});
     });
 
-    it('width should tick thickness with orient vertical', () => {
+    it('has width equals tick thickness with orient vertical', () => {
```

Comment:
```suggestion
    it('have width equal to tick thickness with orient vertical', () => {
```

Same for the ones below.